integrationTests/testProcessorNodeWithMultisigner.go

```diff
@@ -60,7 +60,11 @@ func NewTestProcessorNodeWithCustomNodesCoordinator(
 		accountShardId = 0
 	}
 
-	tpn.OwnAccount = CreateTestWalletAccount(shardCoordinator, accountShardId)
+	if ownAccount == nil {
+		tpn.OwnAccount = CreateTestWalletAccount(shardCoordinator, accountShardId)
```

Comment:
I can't use a setter because right after we set own account, data pools as well as the test node will be initialized and this way wrong single signer and keygen will be used inside interceptors and else where.